src/EventListenerService/Handlers/VoiceStateUpdateHandler.cs

```diff
@@ -62,7 +64,7 @@ public async ValueTask HandleAsync(VoiceState voiceState)
             var usersRemainingInOldChannel = guild.VoiceStates.Values
                                                  .Where(vs => vs.ChannelId == oldChannelId && vs.UserId != voiceState.UserId)
                                                  .Select(vs => vs.UserId);
-            await UpdateChannelStatus(oldChannelId, usersRemainingInOldChannel);
+            await UpdateChannelStatus(scopedWojtusContext, oldChannelId, usersRemainingInOldChannel);
```

Comment:
does it make sense to pass Dictionary<UserId,Emoji> to this method instead of passing the context?